File: MyCourses - Android/app/src/main/java/com/mycoursesapp/model/Booking.java
package com.mycoursesapp.model;

/**
 * Created by mamdouhelnakeeb on 2/14/18.
 */

public class Booking {

    public int id;
    public String centreName;
    public String courseImage;
    public String courseName;
    public String startDate;

    public Booking(String centreName, String courseName, String courseImage, String startDate){
        this.centreName = centreName;
        this.courseName = courseName;
        this.courseImage = courseImage;
        this.startDate = startDate;
    }

    public Booking(int id, String centreName, String courseName, String courseImage, String startDate){
        this.id = id;
        this.centreName = centreName;
        this.courseName = courseName;
        this.courseImage = courseImage;
        this.startDate = startDate;
    }

    public Booking(){}
}

File: MyCourses - iOS/My Courses/Models/Booking.swift
//
//  Booking.swift
//  My Courses
//
//  Created by Mamdouh El Nakeeb on 3/29/18.
//  Copyright © 2018 My Courses. All rights reserved.
//

import Foundation

class Booking{
    
    var id = 0
    var centerName = ""
    var courseImage = ""
    var courseName = ""
    var startDate = ""
    
    init(id: Int, centerName: String, courseImage: String, courseName: String, startDate: String) {
        
        self.id = id
        self.centerName = centerName
        self.courseImage = courseImage
        self.courseName = courseName
        self.startDate = startDate
    }
    
    init(){}
}
